src/test/fixtures/mocks/window-navigation.js

```diff
@@ -3,37 +3,38 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // @flow
-//
 import { coerceMatchingShape } from '../../../utils/flow';
 
+// This symbol will be used in the mock for window.location so that the mock for
+// window.history can change the inner location directly.
+const internalLocationAssign = Symbol.for('internalLocationAssign');
+
 /**
  * jsdom's implementation of this API doesn't allow for assigning to the location, which
  * we need to be able to do for certain tests.
  */
-export function mockWindowLocation(location: string = 'http://localhost') {
+function mockWindowLocation(location: string = 'http://localhost') {
```

Comment:
We don't export individual mocks anymore. Now we export a function that adds them both, and a function that handles beforeEach and afterEach properly that should be easiest way.